.php_cs.dist

```diff
@@ -0,0 +1,37 @@
+<?php
+
+$finder = Symfony\Component\Finder\Finder::create()
+    ->notPath('vendor')
+    ->notPath('tests/fixtures/*')
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ])
+    ->name('*.php')
+    ->notName('*.blade.php')
+    ->ignoreDotFiles(true)
+    ->ignoreVCS(true);
+
+return PhpCsFixer\Config::create()
```

Comment:
If memory serves me right, this is deprecated. Use `new PhpCsFixer\Config` instead